src/device-manager/java/WeaveDeviceManager-JNI.cpp

```diff
@@ -3051,12 +3051,15 @@ WEAVE_ERROR J2N_NetworkInfo(JNIEnv *env, jobject inNetworkInfo, NetworkInfo& out
 
     err = J2N_ByteArrayFieldVal(env, inNetworkInfo, "ThreadExtendedPANId", outNetworkInfo.ThreadExtendedPANId, len);
```

Comment:
Missing from the PR:  modifications to `N2J_NetworkInfo` to account for PSKc, changes to `NetworkInfo.java` to account for a new field, changes to the constructor for above.